test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -33,6 +33,19 @@ class AccountListPage {
       '.multichain-account-menu-popover__list--menu-item-hidden-account [data-testid="account-list-item-menu-button"]';
   }
 
+  async check_pageIsLoaded(): Promise<void> {
```

Comment:
nit: maybe to follow the same as what we did in the [other PR,](https://github.com/MetaMask/metamask-extension/pull/27201/files) I would suggest to place the `checks` at the end

Reply:
oh or maybe is it better to order everything alphabetically? As I see that in other places it is ordered alphabetically including the `check`s. In that case we can leave this as it is, but we can modify the order of the other page (linked PR above), in another PR

Reply:
Thanks for pointing out! What I'm thinking for the order is: `check_pageIsLoaded()` first, followed by `action methods` ordered alphabetically, and then all `check_ methods` ordered alphabetically(as they all start with `check_`).
The reason for placing `check_pageIsLoaded()` first is that all page classes will have this method, and when we create an object for the page, we usually need to call it before performing other actions. Thus, putting this specific method at the beginning will make it easier to find.
What do you think about this approach? If you agree, we can make it a standard.